iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -496,5 +515,23 @@ private static double GenerateTemperatureWithinRange(int max = 50, int min = 0)
         {
             return Math.Round(s_random.NextDouble() * (max - min) + min, 1);
         }
+
+        private async Task UpdateReportedPropertiesWhenWritablePropertiesEmptyAsync(string componentName, CancellationToken cancellationToken)

```

Comment:
UpdateEmptyReportedProperties ? 
This method name is a bit too long.

Reply:
> This method name is a bit too long. `RespondToEmptyWritableProperty` maybe? thoughts?

Sure, I can shorten this and other similar names. What about `UpdateForEmptyWritableProperty` as we don't actually respond to anything here?

Reply:
well, technically you are responding by updating your reported properties. It's considered a response to a writable property request.